packages/server/database/types/MeetingTeamPrompt.ts

```diff
@@ -0,0 +1,30 @@
+import TeamPromptResponsesPhase from './TeamPromptResponsesPhase'
+import Meeting from './Meeting'
+import GenericMeetingPhase from './GenericMeetingPhase'
+
+type TeamPromptPhase = TeamPromptResponsesPhase | GenericMeetingPhase
```

Comment:
+1 Do we need `GenericMeetingPhase` here? Does the team prompt meeting has any of the generic meeting phases?

Reply:
We don't but it's really a side effect of `createNewMeetingPhases` method returning `GenericMeetingPhase[]` not matter what. I don't want to touch this area of code too much as it affects all other meetings so I've added it to silence the TS. Other meetings like retro, do not specify the phase types at all here: https://github.com/ParabolInc/parabol/blob/7c60878ee1740fd9b60a1d62c10b584d04802d11/packages/server/database/types/MeetingRetrospective.ts#L9 so I'm not sure what's the lesser evil ;)

Reply:
sure. if you remove `GenericMeetingPhase` from this line, in `startTeamPrompt` mutation TS will complain because it can't automatically cast `GenericMeetingPhase[]` to `TeamPromptResponsesPhase[]` when creating a new instance of `MeetingTeamPrompt`. 
<img width="980" alt="Screenshot 2022-03-24 at 15 29 23" src="https://user-images.githubusercontent.com/1017620/159938943-e7017333-1a09-4a1e-b229-e77300fa29bc.png">